src/Units/DoRUnits.java

```diff
@@ -31,7 +31,7 @@ public String toString()
   @Override
   public String getIconicUnitName()
   {
-    return "Bike";
+    return "Infantry";
```

Comment:
You added support for loading different unit types, and then changed the iconic type back to infantry?

Reply:
Yeah. Killing the unit role enum entirely meant I couldn't get it from the enum directly, so basing it on the iconic unit was the next logical step. It might make sense to instead just create a set of `UnitModel`s from the UnitSet and just use that, but that feels... hacky?

Based on our previous discussion, it makes sense to use infantry since that's what differentiates most factions anyhow.

Reply:
If we always use Infantry as the iconic type--and it sounds like you don't intend to change this--why keep `getIconicUnitName` and related things?